public/app/features/correlations/Forms/TransformationsEditor.tsx

```diff
@@ -26,21 +15,14 @@ const getStyles = (theme: GrafanaTheme2) => ({
     font-size: ${theme.typography.h5.fontSize};
```

Comment:
can we put these into an object to avoid adding to the betterer warnings? something like: 
```
const getStyles = (theme: GrafanaTheme2) => {
  return {
    heading: css({
      fontSize: `${theme.typography.h5.fontSize}`,
      fontWeight: `${theme.typography.fontWeightRegular}`,
    }),
  };
};
```

Reply:
Good point! Updated in c96d238d6f9e5de7d694178c30c3c947d8b3cac1